engine/lib/river.php

```diff
@@ -868,8 +868,8 @@ function _elgg_river_enable($event, $type, $entity) {
  */
 function _elgg_river_init() {
 	elgg_register_page_handler('activity', '_elgg_river_page_handler');
```

Comment:
If you want to remove the page entirely you would want to get rid of the page handler as well, otherwise it should only be the menu item. Also agree on the deleting instead of commenting out since this is all in git.